src/components/Panel/Panel.tsx

```diff
@@ -28,30 +28,34 @@ export const Panel = React.forwardRef<PanelRef, PanelProps>(
             actionButtonOneProps,
             actionButtonTwoProps,
             actionButtonThreeProps,
-            size = PanelSize.medium,
-            visible = false,
-            closable = true,
-            onClose = () => {},
+            autoFocus = true,
+            bodyClassNames,
+            bodyPadding = true,
             children,
-            placement = 'right',
+            closable = true,
+            closeButtonProps,
+            closeIcon = IconName.mdiClose,
+            footerClassNames,
+            headerButtonProps,
+            headerClassNames,
+            headerIcon = IconName.mdiArrowLeftThick,
```

Comment:
no default header icon

Reply:
This Icon doesn't show unless the headerIconProps are defined.